metagpt/provider/openai_api.py

```diff
@@ -150,6 +149,7 @@ def __init_openai(self, config):
     async def _achat_completion_stream(self, messages: list[dict]) -> str:
         response = await openai.ChatCompletion.acreate(
             **self._cons_kwargs(messages),
+            timeout=3,
```

Comment:
put this into _cons_kwargs

Reply:
> put this into _cons_kwargs

Thanks for reviewing, but is there necessary to put timeout into _cons_kwargs. In my opinion, _cons_kwargs will store params like temperature or model type and so on.

Reply:
or is this will be done?
<img width="1304" alt="image" src="https://github.com/geekan/MetaGPT/assets/12390909/1918ff61-5ec0-48b5-82e5-bab7c4163881">

Reply:
> or is this will be done? <img alt="image" width="1304" src="https://user-images.githubusercontent.com/12390909/258734992-1918ff61-5ec0-48b5-82e5-bab7c4163881.png">

It is done like above~ PTAL